CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.2.1+2
+
+* Update reflectable to work with analyzer 0.39.4 and up to 0.40.0.
```

Comment:
[nit] This line doesn't match the change in the pubspec which has an upper bound of `0.39.4`